test/Microsoft.ML.TimeSeries.Tests/TimeSeriesSimpleApiTests.cs

```diff
@@ -169,7 +169,7 @@ public void SpikeDetection()
         [Fact]
         public void SsaSpikeDetection()
         {
-            var env = new MLContext();
+            var env = new MLContext(1);
```

Comment:
As discussion topic, does any know of a reason to not have a fixed seed for unit tests?